lib/devtools-require.js

```diff
@@ -15,8 +15,8 @@ const ID_RENAMES = [
     replacement: "devtools/client/"
   },
   {
-    regex: /^devtools\/shared\/styleinspector\//,
-    replacement: "devtools/styleinspector/"
+    regex: /^devtools\/shared\/inspector\//,
+    replacement: "devtools/inspector/"
```

Comment:
This replacement is not quite right. We want the path to change from what works for 46 to what works for 45. So, it seems like the replacement value should be `devtools/shared/styleinspector`, since that was the old path.

Also, the original file move that required the first set of replacements here only landed in 44. We need to support all active channels with this, so that's currently 43 - 46. You should leave the previous replacement in place and add a new one just before it. This way all 3 forms of path across 43 - 46 should be supported.